cmd/compare.go

```diff
@@ -130,6 +130,9 @@ func compare(provider string, oldCommit string, newCommit string) error {
 			violations = append(violations, vs...)
 		}
 
+		if f.Outputs == nil {
+			continue
```

Comment:
Here it seems reasonable (though perhaps brittle) to skip the rest of the loop code. Above, examining the outputs is skipped if the inputs are `nil` -- is that definitely correct?

Reply:
Less brittle: put the processing inside `if f.Outputs != nil { ... }`